Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -1116,7 +1115,7 @@ open class PinnedSite: Site {
     let isPinnedSite = true
 
     init(site: Site) {
-        super.init(url: site.url, title: site.title, bookmarked: site.bookmarked)
+        super.init(url: site.url, title: site.title, bookmarked: site.bookmarked, guid: site.guid)
```

Comment:
How will this work for old pinned sites that dont have a guid

Reply:
I checked this in the DB and pinned sites always seem to have a GUID set.